Tensile/TensileCreateLib/ParseArguments.py

```diff
@@ -46,7 +46,7 @@ def splitExtraParameters(par):
     return (key, value)
 
 
-def parseArguments(input: List[str] | None = None) -> Dict[str, Any]:
+def parseArguments(input: Optional[List[str]] = None) -> Dict[str, Any]:
```

Comment:
Is this preferred over |?

Reply:
Some of the images I pulled for testing were still on Python3.6 which doesn't support the `|` operator for type-hints. I changed this to appease these older versions, however, there are still parts of the code that use `|` operators so it may have been a temporary environment issue. I will test the original again in the appropriate images and determine whether this change can be reverted.